fiftyone/brain/similarity.py

```diff
@@ -32,13 +32,21 @@
 
 logger = logging.getLogger(__name__)
 
+_ALLOWED_ROI_FIELD_TYPES = (
+    fol.Detection,
+    fol.Detections,
+    fol.Polyline,
+    fol.Polylines,
+)
+
 _DEFAULT_MODEL = "mobilenet-v2-imagenet-torch"
 _DEFAULT_BATCH_SIZE = None
 
 
 def compute_similarity(
     samples,
     patches_field,
+    roi_field,
```

Comment:
what's the difference between ROI field and patches field from a functional standpoint?

Reply:
So in that case passing a single patch per sample to patches_field is equivalent to ROI field? feels strange to have them as two different arguments.